### PR TITLE
CoffeeToWaterRatio: Convert to Full Template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Re: #2767

Finishing off https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2819 Thanks @lamanh for the work!

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/coffee_to_water_ratio

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nickselpa

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/CoffeeToWaterRatio.pm [hunk 1]
```diff
@@ -12,54 +12,45 @@ zci is_cached   => 1;
...
-my $metric_fluid_units = 'ml';
-
-my $metric_precision =  1;
-my $imperial_precision = 0.1;
-
-my $imperial_to_water = 16;
-my $metric_to_water = 16.6945;
-
-my $MAX_WEIGHT = 100000; # limit to 100,000
+my $MAX_WEIGHT = 100_000;
```
Comment (gaulrobe) [inline lib/DDG/Goodie/CoffeeToWaterRatio.pm]: Being a Perl beginner, this underscore through me off. Pretty neat way to get more readability for larger numbers!

**Comment by GuiltyDolphin** [comment]
@mintsoft @lamanh Thanks! :dolphin:

File: lib/DDG/Goodie/CoffeeToWaterRatio.pm (hunk 1 above)
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/CoffeeToWaterRatio.pm]: Yeah, `1_000_000` :wink:

**Pull request merged**